tests/functional/builtins/codegen/test_create_functions.py

```diff
@@ -146,9 +146,12 @@ def test(_salt: bytes32) -> address:
         c.test(salt, transact={})
 
 
+ERC5202_PREFIX = b"\xFE\x71\x00"
```

Comment:
import from conftest?

Reply:
it should be in vyper/utils.py really

Reply:
https://github.com/vyperlang/vyper/pull/3454/commits/afa1af04e7a98928914f9394d89244fe1b85c0f5